Makefile

```diff
@@ -1,10 +1,17 @@
+RELEASE_FILE = plik-`cat ../VERSION`.tar.gz
+DEST_DIR = /opt/plik
+
 all: clean deps build
 
+install-devtools:
```

Comment:
-1 for me
This depends on a specific user setup
User should be told in the build section of the readme file how to setup a proper build environment